interfaces/matlab/toolbox/@Kinetics/getDeltaEntropies.m

```diff
@@ -0,0 +1,14 @@
+function dH = getDeltaEntropies(a)
+% GETDELTAENTROPIES  Get the entropy of reaction for each reaction.
+% dH = getDeltaEntropies(a)
+%
+% :param a:
+%     Instance of class :mat:func:`Kinetics` (or another
+%     object deriving from Kinetics) for which the entropies of
+%     reaction are desired.
+% :return:
+%     Returns a vector of the entropy of reaction for each
+%     reaction. Units: J/kmol-K
+%
+
+dH = kinetics_get(a.id, 17, 2);
```

Comment:
Sorry, I know I already approved this, but this should probably be dS and the Gibbs should probably be dG

Reply:
I was about to make the same request! :grin:

Reply:
You guys are _so rigid_ in your style preferences. :)  

Fixed now - thanks for the catch!